test/UnitTest/Microsoft.AspNet.OData.Test.Shared/Formatter/Deserialization/DeserializationHelpersTest.cs

```diff
@@ -329,6 +329,66 @@ public void ApplyProperty_DoesNotIgnoreKeyProperty_WithInstanceAnnotation()
             resource.Verify();
         }
 
+
+
+        [Fact]
+        public void ReadResource_DeletedResource_WithTransientTypeAndAnnotations()
+        {
+            // Arrange
+            ODataConventionModelBuilder builder = ODataConventionModelBuilderFactory.Create();
+            builder.EntityType<SimpleOpenCustomer>();
+            builder.EnumType<SimpleEnum>();
+            IEdmModel model = builder.GetEdmModel();
+
+            IEdmEntityTypeReference customerTypeReference = model.GetEdmTypeReference(typeof(SimpleOpenCustomer)).AsEntity();
+            ODataDeserializerProvider _deserializerProvider = ODataDeserializerProviderFactory.Create();
+            var deserializer = new ODataResourceSetDeserializer(_deserializerProvider);
+
+            var instAnn = new List<ODataInstanceAnnotation>();
```

Comment:
```suggestion
            var instanceAnnotations = new List<ODataInstanceAnnotation>();
```
I don't see any reason to shorten a variable name to the point where it's not self-documenting, whether or not it's used in tests